flang/include/flang/Lower/MultiImageFortran.h

```diff
@@ -1,13 +1,13 @@
-//===-- Lower/Coarray.h -- image related lowering ---------------*- C++ -*-===//
+//===-- Lower/MultiImageFortran.h -- image related lowering -----*- C++ -*-===//
 //
 // Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef FORTRAN_LOWER_COARRAY_H
-#define FORTRAN_LOWER_COARRAY_H
+#ifndef FORTRAN_LOWER_MULTI_IMAGE_FORTRAN_H
```

Comment:
nit: The convention is `FORTRAN_LOWER_MULTIIMAGEFORTRAN_H`, even if that is less readable (only folders `/` create `_`.